components/aggregation/mat/Form.js

```diff
@@ -52,14 +52,18 @@ export const Form = ({ onSubmit, testNames, query }) => {
 
   return (
     <form onSubmit={handleSubmit(onSubmit)}>
-      <Flex my={2} alignItems='center'>
-        <Box width={1/3}>
+      <Flex my={2} justifyContent='space-between' sx={{
+        '@media screen and (max-width: 476px)': {
+          flexDirection: 'column',
+        }
+      }}>
+        <Box width={[1, 1/5]}>
           <StyledLabel>
             Country
           </StyledLabel>
           <Controller
             render={({field}) => (
-              <Select {...field}>
+              <Select {...field} sx={{ width: '100%' }}>
```

Comment:
I think it is better to avoid using an object literal to pass down props for these kind of components. It causes unnecessary re-renders when the parent renders for unrelated reasons. You can instead just pass `width={1}` because `Select` is constructed with a `Flex` component.